app/(main)/league/[leagueId]/entry/all/page.tsx

```diff
@@ -137,8 +192,16 @@ const Entry = ({
                 >
                   Add New Entry
                 </Button>
+                {currentWeek > 1 && (
+                  <Link
+                    className="text-orange-600 hover:text-orange-500 font-bold hover:underline"
+                    href={`#`}
```

Comment:
Is this a placeholder href url?

Reply:
Yes it is.

Reply:
> Is this a placeholder href url?

Yes it is as I do not know how we are accessing the previous week's entries.